### PR TITLE
Chore/styling spam banner

### PR DESCRIPTION
**Pull request by MildTomato**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

supabase-spam-assistant

YES

Small update to Table styling

### CODE REVIEW

**Review by alaister — APPROVED** [review]
LGTM!

- Tested on preview

File: apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx [hunk 1]
```diff
@@ -28,33 +38,38 @@ export const SpamValidation = ({ validationResult }: SpamValidationProps) => {
         hasSpamWarning ? (
           <>
```
Comment (joshenlim) [inline apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx:39]: this fragment is probably unnecessarily now

Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx:39]: spamRules.length > 0 has the same check as hasSpamWarning too i realised - will push a refactor

**Review by joshenlim — APPROVED** [review]
LGTM! Verified style changes on staging

**Pull request merged**